src/components/DailyStatistics/DailyStatisticsScreen.tsx

```diff
@@ -94,9 +94,7 @@ const DailyStatisticsScreen: FunctionComponent<NavigationProps> = ({
 
   useEffect(() => {
     if (error) {
-      showErrorAlert(new Error(ERROR_MESSAGE.SERVER_ERROR), () =>
-        navigateHome(navigation)
-      );
+      showErrorAlert(error, () => navigateHome(navigation));
```

Comment:
For server errors, we can `navigateHome` since it might be connectivity related and user can just retry immediately. However for `sessionError`, we should move the user to the `CampaignLocationScreen` instead? Can refer to how it's done for in `CustomerQuotaScreen.ts`
e.g.
```
 if (error instanceof SessionError) {
        clearCartError();
        expireSession();
        showErrorAlert(error, () => {
          navigation.navigate("CampaignLocationsScreen");
        });
        return;
      }
```